### DISCUSSION TITLE
👋 欢迎！新人必读

### DISCUSSION THREAD

**Discussion by 521xueweihan** [521xueweihan/HelloGitHub#1536] [精选]
HelloGitHub 是属于开源爱好者的社区。我们希望你在这里：

* 分享、讨论开源项目
* 与其他社区成员交流
* 铭记，这是我们共同建设的社区 💪

[点击推荐项目到月刊](https://github.com/521xueweihan/HelloGitHub/issues/new)

**Comment by salihukeelahh** [comment]
Hello my name is salihu, i have intrest in learning python from a mentor so if there is any i would love to learn from and love to build more projects in the future 
my details for more contact: salihujr@gmail.com or salihujrr@gmail.com
                                             09032237648

**Reply by Fillare** [reply]
are u really interested in learning python?? I am not a mentor but a student who has learned basic and advance python through python documentation and tutorials. If you think i can be any of help to you ,let me know.

**Reply by caetang** [reply]
> are u really interested in learning python?? I am not a mentor but a student who has learned basic and advance python through python documentation and tutorials. If you think i can be any of help to you ,let me know.
> 
> 

where are you come from，I am learning Python just now.i want to    Learn some experience from you.

**Reply by salihukeelahh** [reply]
yes @Fillare I can really use your help. and by the way am so sorry for late reply, so how can we in contact with each other

**Reply by salihukeelahh** [reply]
@Fillare hey

**Reply by salihukeelahh** [reply]
@caetang am from Nigeria, where are you from? am still leaning python but maybe we can get in contact and share ideas to build a project together. how does that sound

**Reply by Fillare** [reply]
> @Fillare hey

sorry i got into an accident that's why i couldn't reply to you

**Reply by Fillare** [reply]
> > are u really interested in learning python?? I am not a mentor but a student who has learned basic and advance python through python documentation and tutorials. If you think i can be any of help to you ,let me know.
> 
> where are you come from，I am learning Python just now.i want to Learn some experience from you.
i am from Pakistan

**Reply by salihukeelahh** [reply]
ooooh sorry about the accident i hope you are doing better now

**Reply by salihukeelahh** [reply]
am actually interested atleast we can both benefit from each other since we are still learning and we can start building small projects form the scratch

**Comment by caetang** [comment]
am from china. we can get in contact.




| |
tangjie150414
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年12月22日 03:48 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [521xueweihan/HelloGitHub] 👋 欢迎！新人必读 (Discussion #1536) |

@caetang am from Nigeria, where are you from? am still leaning python but maybe we can get in contact and share ideas to build a project together. how does that sound

**Reply by salihukeelahh** [reply]
wow thank you so much. so how we will reach out to each other?

**Comment by StanleyJijieji** [comment]
来催更了！！

**Reply by 521xueweihan** [reply]
@StanleyJijieji 收到，网站内容明天就更！

**Comment by Ck567-sudo** [comment]
所以你喜欢的人叫薛伟涵？

**Reply by 521xueweihan** [reply]
是因为xueweihan被注册了

**Reply by qimingzitainanlea** [reply]
发在这里她又看不到 能运行吗？> 是因为xueweihan被注册了

**Comment by liu-1900** [comment]
yes, I'm interested in python!! But my major is the study of mechanical manufacturing automation, and English is my disadvantage in learning python. I want to learn English through communication. Can you help me
I'm from China
PS: This article was completed with the help of Baidu Translation





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "521xueweihan/HelloGitHub"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年6月19日(星期四) 下午5:08
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [521xueweihan/HelloGitHub] 👋 欢迎！新人必读 (Discussion #1536)





   
are u really interested in learning python?? I am not a mentor but a student who has learned basic and advance python through python documentation and tutorials. If you think i can be any of help to you ,let me know.
  
where are you come from，I am learning Python just now.i want to Learn some experience from you.
 i am from Pakistan

**Comment by Jack-tendy-538** [comment]
I am a Chinese too, but I have to face the pressure of College Entrance Exam.
My master lang is python, but you may see me developing C++ project now.

**Reply by TheForggtenedName** [reply]
哥们这么有实力吗？高中就已经可以开发C++项目了，佩服

**Reply by Jack-tendy-538** [reply]
就是，去年我们学校的学长开了个单片机讲座，虽然我没参加，但是我一个朋友去了，然后我偷偷摸摸的看了几眼他们发的c++教程，刚好学校机房电脑有dev c++，我就稍微实践了一下

**Reply by ling-voc** [reply]
我现在14 前年就已经接触c++了 去年csp j一等s二等 现在在学前后端😇> 哥们这么有实力吗？高中就已经可以开发C++项目了，佩服

**Reply by ling-voc** [reply]
我是在三年级接触了py 五年级开始学c#下半学期开始对c++有兴趣 **立志要当oi爷**😇> 我现在14 前年就已经接触c++了 去年csp j一等s二等 现在在学前后端😇> 哥们这么有实力吗？高中就已经可以开发C++项目了，佩服
> 
>

**Reply by Jack-tendy-538** [reply]
嗯……

**Reply by salihukeelahh** [reply]
yoo guys please speak english

**Reply by Jack-tendy-538** [reply]
@x-ray-os 不要在你那边视奸我
